kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -1023,6 +1023,90 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     message: 'Folders',
     context: "Tab label in the 'Browse channel' page that allows to navigate through its topics.",
   },
+
+  // TODO - move these into diff sections as we make this a full feature in 0.16
+  // Past Papers Project (12/2021) strings
+  scoreLabel: {
+    message: 'Score',
+    context:
+      "In the 'Reports' tab, coaches can see the score that each learner has obtained on a quiz.\n\nThis is represented as a percentage and indicates the questions they've answered correctly.",
+  },
+  timeSpentLabel: {
+    message: 'Time spent',
+    context: 'Indicates the time a learner has taken on a specific learning resource.',
+  },
+  practiceQuizReportTitle: {
+    message: 'Report for {quizTitle}',
+    context: 'Title of the independent practice quiz',
+  },
+  practiceAgainButton: {
```

Comment:
I believe the _Practice_ wording here comes as the parallel to the name **practice quizzes**. 

_Try again_ would be simpler but more generic, and since the string is for a button label used to re-take the quiz, I believe it reinforces the paradigm of _resource-used-for-pracing_ (again and again if needed).

![723efdda6cbce742d8c5fe79022cf5ba](https://user-images.githubusercontent.com/1457929/141387342-c81c4d70-46c6-4982-9987-d343f9364b23.png)

Reply:
@radinamatic @sairina Got it, ty. I'm also curious for @jtamiace thoughts here but to me 'practice **again**' sounds redundant. Maybe the button can simply say `practice`?  That verb already carries the idea of doing something again and again... But obvi I'll defer to you all.

Reply:
Just seeing this - honestly this is just my opinion but in English, "Try again" sounds a lot more natural to me. Would that be a concern for other languages?

Reply:
Not in the 4 languages I speak, no, Jess, quite the contrary I'd say. I think 'Try Again' is universally more broadly used than "Practice Again".

Reply:
Awesome, thank you. Unless there are downsides anyone can think of, can we make this change then? I'll update Figma